src/main/java/org/reaktivity/nukleus/http/internal/stream/ClientConnectReplyStream.java

```diff
@@ -130,7 +130,7 @@ private void handleStreamBeforeBegin(
         }
     }
 
-    private void handleStreamWhenBuffering(
+    private void handleStream(
```

Comment:
Add `BeginFW.TYPE_ID` to `switch`, remove `streamState` and receive all stream frames directly in `handleStream`.